doc/health-check.md

```diff
@@ -78,6 +78,8 @@ contains: `active` or `passive`.
 
 * `active`: To enable active health checks, you need to specify the configuration items under `checks.active` in the Upstream object configuration.
 
+  * `active.timeout`: Socket timeout for active checks (in seconds).
```

Comment:
You should indicate support for decimals.

Reply:
nice, fixed